code-examples/quarkus/kafka-quickstart/src/main/resources/application.properties

```
@@ -0,0 +1,47 @@
+# Quarkus config
+quarkus.ssl.native=true
+
+# Configure the Kafka sink (we write to it)
+mp.messaging.outgoing.generated-price.connector=smallrye-kafka
+mp.messaging.outgoing.generated-price.topic=prices
+mp.messaging.outgoing.generated-price.value.serializer=org.apache.kafka.common.serialization.IntegerSerializer
+
+# Configure the Kafka source (we read from it)
+mp.messaging.incoming.prices.connector=smallrye-kafka
+mp.messaging.incoming.prices.topic=prices
+mp.messaging.incoming.prices.value.deserializer=org.apache.kafka.common.serialization.IntegerDeserializer
+
+# Configure docker config
+quarkus.container-image.builder=docker
+quarkus.kubernetes.deployment-target=kubernetes
+quarkus.container-image.build=true
+quarkus.container-image.push=true
+
+## Service binding configuration used by default for dev and production profiles
+
+quarkus.kubernetes-service-binding.root=/bindings
+
+## dev-env profile using environment variables
+## ./mvnw package -Dquarkus.profile=dev-env
+
+dev-env.kafka.bootstrap.servers=${BOOTSTRAP_SERVER}
```

```suggestion
%dev-env.kafka.bootstrap.servers=${BOOTSTRAP_SERVER}
```

The profile is missing `%` - see https://quarkus.io/guides/config#configuration-profiles

Maybe this can only be `%dev` which is the default `dev` profile when someone does `mvn quarkus:dev` and the rest added profiles can be sepecified via `%my-profile.` and `mvn quarkus:dev -Dquarkus.profile=my-profile`.

---

As @machi1990 says, this should be the built in `%dev` profile.

---

Cool. Adding this now